.travis.yml

```diff
@@ -1,22 +1,16 @@
 language: c
 install:
-  - wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
-  - wget https://raw.githubusercontent.com/simonjbeaumont/ocaml-travis-coveralls/master/travis-coveralls.sh
+  - wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-opam.sh
 script:
     - bash -ex .travis-opam.sh
 sudo: required
 env:
   global:
-    - OCAML_VERSION=4.04
+    - OCAML_VERSION=4.06
     - PACKAGE=message-switch
     - PINS="message-switch-async:. message-switch-cli:. message-switch-core:.  message-switch:.  message-switch-lwt:.  message-switch-unix:."
   matrix:
-    - BASE_REMOTE=git://github.com/xapi-project/xs-opam XS_COV=1 COV_CONF="./configure" \
```

Comment:
Why did we remove this Travis job?

Reply:
It doesn't work anymore. The Travis scripts change, are now using Opam2 and I could it not get to work quickly.